wormhole-foundation/cosmos-sdk
# Update 0.45.9

This updates v0.45.9 which fixes a critical vulnerability in cosmos-sdk.
https://github.com/cosmos/cosmos-sdk/releases/tag/v0.45.9

Redo of https://github.com/wormhole-foundation/cosmos-sdk/pull/100 to make this easier to verify.

To verify:
```
git clone https://github.com/wormhole-foundation/cosmos-sdk
git remote add upstream https://github.com/cosmos/cosmos-sdk
git fetch upstream
git rebase  --committer-date-is-author-date  v0.45.9
git log
```

edit: appears this changed the order of commits, redoing again..
